index.bs

```diff
@@ -1136,11 +1136,11 @@ the user agent must run the steps below.
 The {{FileReader/readAsText()}} method</h5>
 
 The {{FileReader/readAsText()}} method can be called with an optional parameter,
-<dfn argument for="FileReader/readAsText(blob, label), FileReader/readAsText(), FileReaderSync/readAsText(blob, label)" id="dfn-label">label</dfn>,
+<dfn argument for="FileReader/readAsText(blob, encoding), FileReader/readAsText(), FileReaderSync/readAsText(blob, encoding)" id="dfn-encoding">encoding</dfn>,
```

Comment:
I'd somewhat prefer it if we kept the IDs unchanged or at least kept the old IDs working.

Reply:
I can add the legacy reference too. Is there a precedent for keeping legacy links that I can follow in terms of documenting the reasoning, or possibly a Bikeshed tooling support for adding the legacy reference?

If not, I propose to add this line below:

```html
<a id="dfn-label"><!-- Legacy anchor for when argument name was "label" --></a>
```

Reply:
`oldids` as attribute should work, but I'm not sure it's worth changing this at all. I guess the editor can decide.

Reply:
Ah neat! I'll just add that in then since the support already exists. If for any reason (but I don't see one) it should be removed I don't mind removing it again.